requirements.txt

```diff
@@ -1,6 +1,7 @@
 Sphinx==1.3.4
 dj-database-url==0.3.0
 flake8==2.4.0
+inflection==0.3.1
```

Comment:
You should add this to `install_requires.txt`, not `requirements.txt`.

`requirements.txt` is for development requirements (running dynamic-rest as an app or running tests/`make` commands)
`install_requires.txt` is for library requirements. When other apps import dynamic-rest as a dependency, only the dependencies specified in `install_requires.txt` will be installed (via `python setup.py install`)